### DISCUSSION TITLE
Working on New Levels

### DISCUSSION THREAD

**Discussion by KodeBreaker-Has-SWAG** [Neverball/neverball#257] [Show and tell]
Recently I've been making new ideas for some levels. Potentially, I could make a seventh set! It would be harder than Expert, and some of the levels I've made were inspired by SMB. 
Here is what I've made so far:
![Skyscraper](https://user-images.githubusercontent.com/40520550/144076779-094a4529-ba05-4f76-b56f-4e588b3986ec.png)
![Skyscraper2](https://user-images.githubusercontent.com/40520550/144076845-ca38f375-abdb-4d1e-8331-e7c4426352df.png)
![Bracelets](https://user-images.githubusercontent.com/40520550/144076867-ab3fba9a-3ae7-4708-ac6a-a82f7131db68.png)

**Comment by parasti** [comment]
That last one looks insane (in a good way).

**Comment by Adacrop-The-Great** [comment]
All these levels look epic! The seventh set should be called "Kodebreaker Extreme".

**Reply by KodeBreaker-Has-SWAG** [reply]
I already came up with a name, but I won’t spoil it until release ;)

**Reply by Adacrop-The-Great** [reply]
you should move endgame to this new set so it can be preserved as the series conclusion

**Reply by KodeBreaker-Has-SWAG** [reply]
I’m planning to keep Endgame in Expert and have an even more extravagant level for the new set.

**Reply by Adacrop-The-Great** [reply]
Speaking of Expert, I have a suggestion: "Earth Switch" (Expert Level IV) and "Kodebreaker's Maze V" (Expert Level 15) should swap places because "Earth Switch" feels easier to complete than "Kodebreaker's Maze V".

**Reply by KodeBreaker-Has-SWAG** [reply]
I think Earth Switch is good as a bonus level. Plus I like having KB Maze V as a normal level because Expert is a hard set. If the level really is that hard, then I could move it into the last row and put Cutting Corners into the fourth.

**Reply by Adacrop-The-Great** [reply]
that's a better idea actually

**Reply by KodeBreaker-Has-SWAG** [reply]
What's funny is that while making Expert, KB Maze V originally started in the City environment and Cutting Corners started out in the Alien environment

**Reply by Adacrop-The-Great** [reply]
i can totally imagine cutting corners with the alien background, even Reshaun's night alien background.

**Comment by DonAlgo** [comment]
The levels look nice, like, the third one, how it is gonna be? xd

**Reply by KodeBreaker-Has-SWAG** [reply]
Don’t want to spoil how the levels will be. You’ll have to see when you play it yourself 😉

**Comment by KodeBreaker-Has-SWAG** [comment]
The third one I'm planning to buff it since it's a little _too_ easy for this set

**Reply by Adacrop-The-Great** [reply]
One way to buff it is by making the rings narrower.

**Reply by KodeBreaker-Has-SWAG** [reply]
That's what I was planning. I make the rings more narrow as the level progresses. I was looking at five or six levels that could easily be the first six levels. That's when I realized that I should probably buff some of these because I don't want to start the set way too easy; especially compared to how Expert starts.
